src/normalizations/normalizations-utils.ts

```diff
@@ -80,13 +81,17 @@ export const getAlarmToString = (alarm: any): any => {
 
 		if (rel) {
 			let [number, unit] = [null, null];
+
 			const seconds =
 				(rel.s || 0) +
 				(rel.m || 0) * SECONDS_PER_MINUTE +
 				(rel.h || 0) * SECONDS_PER_MINUTE * MINUTES_PER_HOUR +
 				(rel.d || 0) * SECONDS_PER_MINUTE * MINUTES_PER_HOUR * HOUR_PER_DAY +
 				(rel.w || 0) * SECONDS_PER_MINUTE * MINUTES_PER_HOUR * HOUR_PER_DAY * DAY_PER_WEEK;
 
+			if ((rel.s || 0) + (rel.m || 0) + (rel.h || 0) + (rel.d || 0) + (rel.w || 0) === 0) {
+				return 'At the time of event';
```

Comment:
This needs to be translated

Reply:
@nubsthead it's not a part of my task and will take time.So I'll ask alice for a separate task for it